gen/templates/resource_test.go.tmpl

```diff
@@ -390,24 +454,51 @@ func TestAccResource{{.resourceClassName}}(t *testing.T) {
 }
 {{end}}
 
+{{- if ne .targets nil }}
+	{{- range .targets}}
+		{{- if ne .parent_dependency "" }}
+const testConfig{{.class_name}}MinDependencyWith{{capitalize .parent_dependency}} = `
+resource "aci_{{$.targetResourceName}}" "test" {
+  parent_dn = {{.parent_dependency_dn}}
+  {{- range $key, $value := .resource_required}}
+  {{$key}} = "{{$value}}"
+  {{- end}}
+}
```

Comment:
Can you adjust the indentation of the templates so that templates become easier to read?
```suggestion
{{- if ne .targets nil }}
	{{- range .targets}}
		{{- if ne .parent_dependency "" }}
const testConfig{{.class_name}}MinDependencyWith{{capitalize .parent_dependency}} = `
resource "aci_{{$.targetResourceName}}" "test" {
  parent_dn = {{.parent_dependency_dn}}
                       {{- range $key, $value := .resource_required}}
  {{$key}} = "{{$value}}"
                       {{- end}}
}
```